File: docs/code_examples/playwright/javascript/screenshots_pdf.js
import { chromium } from 'playwright';
import fs from 'fs';

const browser = await chromium.connect({
    wsEndpoint: 'wss://playwright.xclouds.dev/v1.58/?api_key=YOUR_API_KEY',
});
const page = await browser.newPage();
await page.goto('https://example.com');

const pngBuffer = await page.screenshot();
fs.writeFileSync('screenshot.png', pngBuffer);

const pdfBuffer = await page.pdf({ format: 'A4' });
fs.writeFileSync('page.pdf', pdfBuffer);

await browser.close();

File: docs/code_examples/playwright/csharp/screenshots_pdf.cs
using Microsoft.Playwright;

using var playwright = await Playwright.CreateAsync();
var browser = await playwright.Chromium.ConnectAsync(
    "wss://playwright.xclouds.dev/v1.58/?api_key=YOUR_API_KEY"
);
var page = await browser.NewPageAsync();
await page.GotoAsync("https://example.com");

await page.ScreenshotAsync(new PageScreenshotOptions { Path = "screenshot.png" });
await page.PdfAsync(new PagePdfOptions { Path = "page.pdf" });

await browser.CloseAsync();
